learning_wavelets/models/learnlet_layers.py

```diff
@@ -172,7 +186,9 @@ def call(self, image):
         wav_coarse = wav_coeffs[-1]
         outputs_list = []
         for i_scale, wav_detail in enumerate(wav_details):
-            details_tiled = self.convs_detail_tiling[i_scale](wav_detail)
+            details_tiled_fixed = self.convs_detail_tiling_fixed[i_scale](wav_detail)
+            details_tiled_train = self.convs_detail_tiling_train[i_scale](wav_detail)
+            details_tiled = Concatenate()([details_tiled_fixed, details_tiled_train])
```

Comment:
do not use the layer `Concatenate` here: it will create a layer at inference time which we want to avoid esp. when running in eager mode.

I think it's best to use `tf.concat`

Reply:
same for below

Reply:
solved